sway-fmt-v2/src/utils/item.rs

```diff
@@ -14,6 +14,8 @@ impl Format for Item {
             Abi(item_abi) => item_abi.format(formatter),
             Const(item_const) => item_const.format(formatter),
             Storage(item_storage) => item_storage.format(formatter),
+            Break(_item_break) => unimplemented!(),
```

Comment:
I don't recall if this macro panics. Iirc then there is a newer (shorter) `todo!()` which will panic when reached. I think it was meant to replace this more verbose macro.

Anyway, does this mean that is someone uses these keywords and attempts to format their code then it will panic? That is not ideal if true. I'd rather get a message  informing me of this rather than crashing - unless this only produces a warning which is fine

Reply:
I'm not sure which macro is preferred honestly, but I opened https://github.com/FuelLabs/sway/issues/2118 and I'm hoping we don't release `sway-fmt-v2` without this fixed. cc @eureka-cpu.

Reply:
It doesn't make that much difference, they both panic.  `todo!()` is meant to convey the meaning that this will definitely be done soon, but `unimplemented!()` can mean we'll never do this...